src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -34,41 +32,6 @@
 
```

Comment:
Try to add some testcases to ensure 100% or close to that of method and branch coverage for the EditCommand class.

Reply:
I added it back👍